docs/source/plugins/99_1_solver_api_reference.rst

```diff
@@ -163,6 +163,11 @@ ALFAsim's Solver Data
 
 .. doxygenfunction:: get_flow_pattern
 
+.. doxygenfunction:: set_warning_message
+
+.. doxygenfunction:: set_information_message
+
```

Comment:
I would create a new section in the documentation, something like:: `Solver Logging` or just `Logging` or `Warning and Information Logging`. Also, locate this section right after `Liquid-Liquid Mechanistic Model helpers` Section.

If you think that the section needs a short description, feel free to add one.

Reply:
sure, done